cosmos-sdk-proto/Cargo.toml

```diff
@@ -23,3 +23,7 @@ version = "0.17"
 [features]
 default = ["grpc"]
 grpc = ["tonic"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
This is a bit magical, but it allows passing a `cfg` attribute to rustdoc when building documentation on docs.rs. See immediately below for how it works.